src/build.rs

```diff
@@ -47,6 +48,42 @@ fn rustc_minor_version() -> Option<u32> {
     otry!(pieces.next()).parse().ok()
 }
 
+/// Checks and returns local and latest versions of wasm-pack
+pub fn check_wasm_pack_versions() -> Result<(String, String), Error> {
+    match wasm_pack_local_version() {
+        Some(local) => {
+            match Crate::return_wasm_pack_latest_version() {
+                Some(latest) => Ok((local, latest)),
+                None => Ok((local, "".to_string()))
+            }
+        },
+        None => bail!("We can't figure out what your wasm-pack version is, make sure the installation path is correct.")
+    }
+}
+
+fn wasm_pack_local_version() -> Option<String> {
+    macro_rules! otry {
```

Comment:
🍟 Small fries but the macro feels like overkill to me here 🍟. As I was reading this over I was confused about what otry was until I came and found it.

Reply:
This is also unnecessary these days, since `?` can work with options, e.g the following compiles:

```rust
pub fn foo(maybe_x: Option<u32>) -> Option<String> {
    let x = maybe_x?;
    Some(x.to_string())
}
```

But yeah, what we should really be doing here is including the version in the binary like `env!("CARGO_MANIFEST_VERSION")` as suggested elsewhere.